src/nccl_ofi_rdma.cpp

```diff
@@ -437,15 +438,7 @@ static int write_topo_file(nccl_ofi_topo_t *topo)
 	}
 
 	/* Set topology file path environment variable `NCCL_TOPO_FILE` */
-	NCCL_OFI_INFO(NCCL_INIT | NCCL_NET,
-		      "Setting NCCL_TOPO_FILE environment variable to %s",
-		      filename);
-	if (setenv("NCCL_TOPO_FILE", filename, 1) != 0) {
-		NCCL_OFI_WARN("Unable to set NCCL_TOPO_FILE. ERROR: %s",
-			      strerror(errno));
-		ret = -errno;
-		goto error;
-	}
+	env_manager::getInstance().insert_envvar("NCCL_TOPO_FILE", filename, true);
```

Comment:
The `NCCL_OFI_INFO` log for setting `NCCL_TOPO_FILE` was removed, would we want to keep it?